clusters/clusters_api.go

```diff
@@ -444,6 +444,19 @@ func (cluster Cluster) Validate() error {
 	return fmt.Errorf("NumWorkers could be 0 only for SingleNode clusters. See https://docs.databricks.com/clusters/single-node.html for more details")
 }
 
+// ResolveComputedAttributesForAPIRequest prepocesses the cluster configuration to remove conflicting computed attributes before we can make an API request.
+func (cluster *Cluster) ResolveComputedAttributesForAPIRequest() {
```

Comment:
Does this need to be public?
```suggestion
func (cluster *Cluster) resolveComputedAttributesForAPIRequest() {
```

Reply:
I think so, because this is in the `clusters` package, and I'm using it in `jobs`?

Reply:
You're absolutely correct, my mistake.